src/Scan/RestScans.ts

```diff
@@ -117,15 +120,28 @@ export class RestScans implements Scans {
     };
   }
 
-  private exploreDiscovery(body: ScanConfig): Discovery[] {
+  private async exploreDiscovery(body: ScanConfig): Promise<Discovery[]> {
     const discoveryTypes: Discovery[] = [];
+    const fileId = body.fileId;
```

Comment:
Use destructuring assignment instead
```suggestion
    const { fileId, crawlerUrls } = body;
```